tests/integration/testcases.py

```diff
@@ -38,11 +38,15 @@ def create_service(self, name, **kwargs):
         if 'command' not in kwargs:
             kwargs['command'] = ["top"]
 
+        client = self.client
+        if 'client' in kwargs:
+            client = kwargs.pop('client')
```

Comment:
We're going to be upgrading the minimum API version to 1.21 pretty soon, so we'll be able to drop this.  Tests will fail for now against the older version.  

Once #2421 is merged, we'll only test against 1.21, and I think that branch is a pre-req for these changes so that we can apply this to only the v2 config.